lib/wallaby/dsl.ex

```diff
@@ -1,5 +1,26 @@
 defmodule Wallaby.DSL do
-  @moduledoc false
+  @moduledoc """
+  Sets up the Wallaby DSL in a module.
+
+  All functions in `Wallaby.Browser` are now accessible without a module name
+  and `Wallaby.Browser`, `Wallaby.Element` and `Wallaby.Query` are all aliased.
+
+  ## Example
+
+  ```
+  defmodule MyPageObject do
```

Comment:
I know that coming from Ruby one tends to call them page objects, but they aren't really as it's just a module. I would just call it `MyPage` - what do you think?